### DISCUSSION TITLE
Bridge Authorization to Kafka Cluster

### DISCUSSION THREAD

**Discussion by nnellanspdl** [strimzi/strimzi-kafka-operator#7129] [Q&A]
Say my Kafka cluster is configured with `simple` authorization.

How do I tell Bridge to use a particular username when talking with the Kafka cluster?

**Comment by scholzj** [comment]
You can just configure it in the `KafkaBridge` custom resource. Check the [docs (the `.spec.authentication` section)](https://strimzi.io/docs/operators/latest/full/configuring.html#proc-config-kafka-bridge-str) or the [security examples](https://github.com/strimzi/strimzi-kafka-operator/tree/main/examples/security).
